assets/src/pages/webrtcInternals/parsIncomingStats.tsx

```diff
@@ -0,0 +1,91 @@
+import moment from "moment";
+import { MAX_DATA_POINTS_ON_CHART } from "../room/consts";
+import { tail } from "ramda";
+
+export type Section = {
+  key: string;
+  descriptive: DescriptiveValue[];
+  charts: ChartEntry[];
+  subsections?: { [name: string]: Section };
+};
+
+type DescriptiveValue = {
+  name: string;
+  value: string;
+};
+
+type ChartEntry = {
+  chartTitle: string;
+  xs: number[];
+  ys: number[];
+};
+
+type ChannelInputValue = string | number | ChannelInput | null;
+
+type ChannelInput = {
+  [key: string]: ChannelInputValue;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isChannelInput = (stats: any): stats is ChannelInput => {
+  return stats && typeof stats === "object" && Object.keys(stats).length > 0;
+};
+
+const isCertainTuple = <T extends ChannelInputValue>(
+  stats: [string, ChannelInputValue],
+  typeCheck: (value: ChannelInputValue) => boolean
+): stats is [string, T] => {
+  const [_key, value] = stats; //eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
Shouldn't we configure the rule to allow unused var if it's prefixed with an underscore?